engine/tester/config_parse.c

```diff
@@ -206,7 +206,7 @@ get_leading_space_num(const char *line, const char **new_line)
     if (end == NULL)
         end = line + strlen(line);
 
-    if (line + spaces_cnt == end - 1)
+    if (line + spaces_cnt == end)
```

Comment:
Why is it changed here?
It looks there is something wrong with split of patches.